app.py

```diff
@@ -92,10 +93,28 @@ def main():
         logger.info(data)
         return jsonify(data)
 
+    if request.args.get('method') == "displayboard":
+        try:
+            start = datetime.datetime.now()
+            params = request.args
+            chrome_driver = create_driver(__location__=None)  # open browser
```

Comment:
formatter is not owkring on your Editor. __location__ = None
spaces should be there